.circleci/config.yml

```diff
@@ -1037,14 +1037,16 @@ jobs:
       - run:
           name: test:e2e:chrome
           command: |
-            if .circleci/scripts/test-run-e2e.sh
-            then
-              timeout 20m yarn test:e2e:chrome --retries 2 || echo "Temporarily suppressing MV3 e2e test failures"
+            export ENABLE_MV3=true
```

Comment:
Question: if we set this value as true in `builds.yml`, i assume we could remove this config, is this correct?

Reply:
yeah, but we will set it here first, so that we can get to a place where this is passing on `develop`, and then we will merge your PR where we can remove this. But we shouldn't move your PR until we see `test-e2e-chrome-mv3` passing on `develop`

Reply:
yes definitely makes sense